types/index.d.ts

```diff
@@ -2,1360 +2,1472 @@ declare var RA: RamdaAdjunct.Static;
 // TypeScript Version: 2.4
 
 declare namespace RamdaAdjunct {
```

Comment:
This whole file have been changed. We only want changes regarding `skipTake`.

Reply:
fixed